potatobot/potatobot.py

```diff
@@ -143,6 +157,9 @@ def _should_ignore_post(self, post):
                                  post["nr"]
                                  ))
                 return True
+        for i in post['change_log']:
```

Comment:
If you need to do this answer check in more than one place (above and here) then it should be in a helper function.